### ISSUE TITLE
Off-by-one bug in the definition of PAT_FRL2

### ISSUE THREAD

**Issue by GNiklasch** [GNiklasch/formic-windmill#20]
How did I never notice this before? The color ranges in the left column of `PAT_FRL2` have been off by one all this time:
```javascript
var PAT_FRL2 = [
    LCR_FRLL2,  LCL_RL0,    LCL_RM0,
    LCR_FRLL1,  LCL_RL2,    LCL_RM2,
    LCR_FRLL0,  LCL_RL1,    LCR_GRM1
];
```
Should be `LCR_FRLL0` in the top left corner, and so on down the column.

**Comment by GNiklasch** [comment]
Fixed in master by 2151b79.  Will be in the next release (no timeline yet).

**Comment by GNiklasch** [comment]
Fixed in v1.4 .
